src/sensors/tests/test_distant.py

```diff
@@ -0,0 +1,219 @@
+import numpy as np
+import pytest
+
+import enoki as ek
+import mitsuba
+
+
+def xml_sensor(direction=None, target=None, xpixel=1):
+    if direction is None:
+        xml_direction = ""
+    else:
+        if type(direction) is not str:
+            direction = ",".join([str(x) for x in direction])
+        xml_direction = f"""<vector name="direction" value="{direction}"/>"""
+
+    if target is None:
+        xml_target = ""
+    else:
+        if type(target) is not str:
+            target = ",".join([str(x) for x in target])
+        xml_target = f"""<point name="target" value="{target}"/>"""
+
+    xml_film = f"""<film type="hdrfilm">
```

Comment:
Would be great to use `load_dict` here ;)

Reply:
I updated the tests to use `load_dict`.